iso15118/shared/security.py

```diff
@@ -125,8 +126,16 @@ def get_ssl_context(server_side: bool) -> Optional[SSLContext]:
          Need to figure out a way to securely store those certs and keys
          as well as read the password.
     """
-    # TODO In ISO 15118-20, we use TLS 1.3. Need to adapt that later
-    ssl_context = SSLContext(protocol=PROTOCOL_TLSv1_2)
+
+    if ENABLE_TLS_1_3:
+        ssl_context = ssl.SSLContext()
```

Comment:
creating the context without specifying the protocol is deprecated. select ssl.PROTOCOL_TLS instead

Reply:
Oops...changed now...